site/docs/hive.md

```diff
@@ -0,0 +1,62 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Hive
+
+## Hive read support
+Iceberg supports the reading of Iceberg tables from [Hive](https://hive.apache.org) by using a [StorageHandler](https://cwiki.apache.org/confluence/display/Hive/StorageHandlers). 
+
+### Table creation
+This section explains the various steps needed in order to overlay a Hive table "on top of" an existing Iceberg table.
+
+#### Create an Iceberg table
+The first step is to create an Iceberg table using the Spark/Java/Python API. For the purposes of this documentation we will assume that the table is called `table_a` and that the base location of the table is `s3://some_bucket/some_path/table_a`.
```

Comment:
In reality this is Catalog specific. Works with HadoopTables / HadoopCatalog.
I am not entirely sure, but I think this step could be omitted with HiveCatalog. @marton-bod?

Reply:
I agree, it would be great to have a section for Hadoop path-based tables and a section for Hive catalog tables.

Reply:
Yup, makes sense, I've only covered the HadoopTables path here. I'll look into how to add the others.

Reply:
OK, I've clarified that this document only covers the HadoopTables path. As discussed in the community sync today we agreed it would be better to have *some* documentation, even if we just cover this path for now, and we'll add in documentation for the Hive/Hadoop Catalog paths later.